iOSDFULibrary/Classes/Utilities/ZipArchive.swift

```diff
@@ -21,6 +21,7 @@
 */
 
 import ZIPFoundation
+import Foundation
```

Comment:
@DevAndArtist this entire diff looks amazing! Thank so much!
Could you explain why you added the `Foundation` import here? It doesn't seem like we need it if there are no other changes to this file.

Reply:
This is the only "blocking" comment.

Reply:
In an Xcode framework the import is implicitly, but in SPM you need to be a little bit more explicit what you want to use and where the compiler has to lookup the code. SPM complained that this particular file used unknown files such as `Data`. Therefore I added an explicit import of `Foundation`.

Reply:
@dinesharjani it wasn't much. Again as I mentioned above I would appreciate if we could call the library more like other Swift libraries. `NordicDFU` seems like a good candidate. Furthermore we could consider to split the package to provide two different products.

- `NordicDFU` without a dependency to `ZIPFoundation` (for example in our use of `iOSDFULibrary` we don't need the archiving part at all and bundling an extra library with our product seems unnecessary)
- `NordicDFUExtension` a convenient extension of the library that adds support for zip archives.

Reply:
But we can do this inclemently if you want. ;)